react/finger-exercises/ecmascript/02 - promises/test.js

```diff
@@ -1,18 +1,19 @@
 import { delay, asyncDelay } from '.';
 
 const TIME = 500;
+export default TIME;
```

Comment:
In some files you will want to export several things. So you can export like this

export const TIME = 500

And in the importing file you can import like this
`import { TIME } from './test';`